service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FailedPBAPaymentEventHandler.java

```diff
@@ -68,21 +69,20 @@ private void notifyLocalAuthority(FailedPBAPaymentEvent event, String template)
         notificationService.sendEmail(template, emails, parameters, caseData.getId().toString());
     }
 
-    private void notifyRespondent(FailedPBAPaymentEvent event, Collection<String> emails) {
+    private void notifyRespondent(FailedPBAPaymentEvent event, String email) {
         FailedPBANotificationData parameters = notificationContent
             .getApplicantNotifyData(event.getApplicationTypes(), event.getCaseData().getId());
 
         notificationService.sendEmail(INTERLOCUTORY_PBA_PAYMENT_FAILED_TEMPLATE_FOR_APPLICANT,
-            emails, parameters, event.getCaseData().getId().toString());
+            email, parameters, event.getCaseData().getId().toString());
     }
 
-    private Map<String, Collection<String>> getRespondentsEmails(CaseData caseData) {
-        Map<String, Collection<String>> respondentEmails = new HashMap<>();
+    private Map<String, String> getRespondentsEmails(CaseData caseData) {
+        Map<String, String> respondentEmails = new HashMap<>();
 
-        IncrementalInteger i = new IncrementalInteger(1);
         caseData.getAllRespondents().forEach(respondent -> respondentEmails.put(
-            respondent.getValue().getParty().getFullName() + ", Respondent " + i.getAndIncrement(),
-            Set.of(respondent.getValue().getSolicitor().getEmail())));
+            respondent.getValue().getParty().getFullName(),
```

Comment:
could be just

    private Map<String, String> getRespondentsEmails(CaseData caseData) {
        return caseData.getAllRespondents().stream()
            .map(Element::getValue)
            .filter(respondent -> nonNull(respondent.getSolicitor()))
            .collect(toMap(res -> res.getParty().getFullName(), res -> res.getSolicitor().getEmail()));
    }